docs/book/src/topics/bring-your-own-aws-infrastructure.md

```diff
@@ -142,7 +142,7 @@ To specify additional security groups for the control plane load balancer for a
 ```yaml
 spec:
   controlPlaneLoadBalancer:
-    AdditionalsecurityGroups:
+    additionalsecurityGroups:
```

Comment:
```suggestion
    ingressRules:
```

This section is for the load balancer.

Reply:
additionalSecurityGroups exists in load balancer spec

Reply:
You are right sorry, I was mixing security groups and ingress rules here.